Documentation/compatibility/SignedXml.GetPublicKey-returns-RSACng-on-net462.md

```diff
@@ -0,0 +1,28 @@
+## SignedXml.GetPublicKey returns RSACng on net462 (or lightup) without retargeting change
+
+### Scope
+Edge
+
+### Version Introduced
+4.6.2
+
+### Version Reverted
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+We let an object return type change in net462 without quirking it. This change adds a quirk to use the old method of getting the return value (certificate.PublicKey.Key) instead of the new method (certificate.GetAnyPublicKey)
```

Comment:
This is completely unclear. What changed? What is an "object return type change"? (Objects don't have return types.) The documentation shows no GetAnyPublicKey method, even for 4.7.1.

Reply:
>This is completely unclear. What changed? What is an "object return type change"? (Objects don't have return types.)

The concrete type of the object returned by the GetPublicKey method changed (without a quirk) from a CryptoServiceProvider implementation to a Cng implementation in 4.6.2.
 
This is because the implementation changed from using 
[certificate.PublicKey](https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.x509certificate2.publickey(v=vs.110).aspx).Key to using the internal certificate.GetAnyPublicKey in 4.6.2 which forwards e.g. to [RSACertificateExtensions.GetRSAPublicKey](https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.rsacertificateextensions.getrsapublickey(v=vs.110).aspx)

I'll add some more details.